common/api/core-backend.api.md

```diff
@@ -1726,6 +1735,8 @@ class Element_2 extends Entity {
     removeUserProperties(nameSpace: string): void;
     // @beta
     static readonly requiredReferenceKeys: ReadonlyArray<string>;
+    // @beta
+    static readonly requiredReferenceKeyTypeMap: Record<string, ConcreteEntityTypes>;
```

Comment:
beta method uses alpha type ConcreteEntityTypes. api-extractor doesn't detect such errors cross-package.

Reply:
I will make it alpha

Reply:
done

Reply:
According to core-common.api.md, ConcreteEntityTypes is still `@alpha`.

Reply:
Ah. You switched this from beta to alpha instead.
(Why is ConcreteEntityTypes alpha?)

Reply:
IMO alpha should almost never be used. But I'm not going to further hold up this PR for it.

Reply:
I will keep that in mind for the future then